src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c

```diff
@@ -2232,7 +2240,7 @@ static int xmc_probe(struct platform_device *pdev)
 	platform_set_drvdata(pdev, xmc);
 
 	for (i = 0; i < NUM_IOADDR; i++) {
-		if ((i == IO_CLK_SCALING) && !xmc->runtime_cs_enabled)
+		if ((i == IO_CLK_SCALING) && xmc->runtime_cs_enabled)
```

Comment:
don't follow the logic here. if cs enabled, you don't map the register map?

Reply:
Taken care of this.